cas-server-webapp/src/main/resources/messages.properties

```diff
@@ -20,60 +20,67 @@
 #
 
 screen.welcome.welcome=Congratulations on bringing CAS online!  To learn how to authenticate, please review the default authentication handler configuration.
-screen.welcome.security=For security reasons, please Log Out and Exit your web browser when you are done accessing services that require authentication!
-screen.welcome.instructions=Enter your Username and Password
+screen.welcome.security=For security reasons, please log out and exit your web browser when you are done accessing services that require authentication!
```

Comment:
In this file, updated all the messages, labels, etc., proposed by the Production and QA team.